src/services/index.js

```diff
@@ -0,0 +1,3 @@
+export const GRANT_TYPE = 'password';
+export const CLIENT_ID = 9;
+export const CLIENT_SECRET = '1le9ILvz6HyfVjwB40Kin61SuCDoXhvgFmLmtkLp';
```

Comment:
consider make this available through an .env file to not keep this accesible or if we need to change keys between environments(not mandatory). It is just how to big applications works

Reply:
i don't know how to import these CONST from de .env file.

Reply:
Perfect, it is not a mandatory thing, we can take a look on that in a separate project